pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -78,17 +79,17 @@ def __init__(
         channel_columns: list[str] = Field(
             min_length=1, description="Column names of the media channel variables."
         ),
-        adstock_max_lag: int = Field(
-            ...,
-            gt=0,
-            description="Number of lags to consider in the adstock transformation.",
-        ),
         adstock: str | InstanceOf[AdstockTransformation] = Field(
             ..., description="Type of adstock transformation to apply."
         ),
         saturation: str | InstanceOf[SaturationTransformation] = Field(
             ..., description="Type of saturation transformation to apply."
         ),
+        adstock_max_lag: int | None = Field(
+            None,
+            gt=0,
+            description="Number of lags to consider in the adstock transformation.",
+        ),
```

Comment:
If we have a default of `None` we must ensure in the docstrings and the Field description what is the max lag used. From what I see below is `10` but this has to be explicit for the user in the docs :)